### ISSUE TITLE
The -o flag does't work (anymore) for transaction list command?

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#218]
While I was testing for the #192 (so the address `n29mSoozF9bDBAJgHKdgyyK5GZcDDfyhvm` was already cached till the block 1000) I've run  `transaction list -x -o n29mSoozF9bDBAJgHKdgyyK5GZcDDfyhvm -l -f 1 -e 1000` and got the following message in the output:
```
Locator mode not supported if no addresses or decks are selected. Locators will not be used.
```
And the code has begun processing blocks after that.

What have I been doing wrong in this case?

**Comment by d5000** [comment]
Very stupid bug (wrong variable used when I fixed another thing). Fixed. Will be uploaded together with the other locator fixes. Does not need testing as it was obvious.

**Comment by buhtignew** [comment]
I've run ` address cache n29mSoozF9bDBAJgHKdgyyK5GZcDDfyhvm -e -f`, ` address cache n29mSoozF9bDBAJgHKdgyyK5GZcDDfyhvm -b 1000` and `transaction list -x -o n29mSoozF9bDBAJgHKdgyyK5GZcDDfyhvm -l -f 1 -e 1000` and got the following error message:
```
  File "~/.local/bin/pacli", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/__main__.py", line 494, in main
    fire.Fire({
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 143, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 477, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
                                ^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/fire/core.py", line 693, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/extended_classes.py", line 1783, in list
    return ei.run_command(self.__list, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/extended_interface.py", line 35, in run_command
    result = c(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/extended_classes.py", line 1845, in __list
    txes = bx.show_txes(sending_address=origin, receiving_address=address_or_deck, start=from_height, end=end_height, coinbase=view_coinbase, advanced=advanced, quiet=quiet, debug=debug, burns=False, use_locator=locator)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/blockexp.py", line 121, in show_txes
    blockdata = bu.show_txes_by_block(receiving_addresses=receiving_addresses, sending_addresses=sending_addresses, advanced=advanced, startblock=startblock, endblock=endblock, coinbase=coinbase, quiet=quiet, debug=debug, use_locator=use_locator, locator=locator, store_locator=use_locator)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "~/.local/lib/python3.12/site-packages/pacli/blockexp_utils.py", line 99, in show_txes_by_block
    min_height = min(blockheights)
                 ^^^^^^^^^^^^^^^^^
ValueError: min() iterable argument is empty
```

**Comment by d5000** [comment]
Confirmed. This was a completely different bug though and unrelated to `-o`, happened that if you retrieved blocks in the already cached range but no transactions were found in the caching process. Fixed in commit 8e71bd6.

**Comment by buhtignew** [comment]
It works, closing.
